scripts/build/testdata/dry_run_android-arm64-chip-tool.txt

```diff
@@ -0,0 +1,32 @@
+# Commands will be run in CHIP project root.
```

Comment:
Why are these `.txt` files and not `.sh`?

Reply:
These are test validation data. They are meant as 'sh' when they are executed because they list what the tool would execute, but in the end it is what gets printed to stdout by the tool. SH seems to somehow indicate they would be executable when in fact they would not be.

if sh, I would need a `#!/usr/bin/bash` or similar, would rather not go that path.